examples/accounts/accountDataFetchers.tsx

```diff
@@ -0,0 +1,13 @@
+import type { AuthorizedParty } from '../../lib/hooks/useAccountSelector';
+import * as data from './authorizedPartiesData.json';
+
+export const getAuthorizedPartiesData = (): AuthorizedParty[] => {
```

Comment:
This is a bit fishy because it always trust the content of authorizedPartiesData to AuthorizedParty[],  but understand the trade off, but can this be as simple as this?

```
import type { AuthorizedParty } from '../../lib';
import data from './authorizedPartiesData.json';

export const getAuthorizedPartiesData = (): AuthorizedParty[] => {
  // Handle both default export and direct array import
  return data as unknown as AuthorizedParty[];
};

```